### ISSUE TITLE
Screen readers (NVDA/JAWS) are not narrating role and sorting information for the column headers, while navigating using table short-cut keys (Ctrl+Alt+arrow keys).

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#574]
Impact on a Person with Disability (PWD):
Screen reader users are not aware of the information asScreen readers (NVDA/JAWS) are not narrating role and sorting information for the column headers, while navigating using table short-cut keys (Ctrl+Alt+arrow keys).

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Screen Reader: NVDA (2023.3), JAWS (2022.2110.6)

Pre- Requisite: 
Turn on NVDA/JAWS

Steps to Reproduce:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to the 'History' tab and invoke it.
Step 4: Verify whether Screen readers (NVDA/JAWS) are narrating role and sorting information for the column headers, while navigating using table short-cut keys (Ctrl+Alt+arrow keys) or not.

Actual Result: 
Screen readers (NVDA/JAWS) are not narrating role and sorting information for the column headers, while navigating using table short-cut keys (Ctrl+Alt+arrow keys).
Observation:
i. When focus lands on sortable column header 'Retrospective name', Screen readers are narrating as 'Clickable column 2 Retrospective name' then upon pressing enter key to sort the table, screen reader is remaining silent.
ii. When focus lands on already sorted column header 'Retrospective name', screen readers are narrating as 'Clickable column 2 Retrospective name' 

Expected Result:
Screen readers (NVDA/JAWS) should narrate role and sorting information for the column headers, while navigating using table short-cut keys (Ctrl+Alt+arrow keys).
Ex: 
i. when focus is on sortable column header 'Retrospective name', screen reader should narrate as "column 2 Retrospective name' button".
ii. When focus lands on sortable column header 'Retrospective name' which is already sorted, then NVDA should narrate as "column 2 Retrospective name button sorted ascending/descending".
iii. When focus lands on any of the sortable column headers, on invoking enter to sort the column, focus should be on the sorted control itself and should narrate sorting information as "sorted ascending/descending".
Ensure that: Sortable chevron icons should be provided for the sortable column headers and should have the contrast ratio of 3:1 with the background.

Suggestive Fix: 
1.Provide 'role="button" for the column header.
2.Provide 'aria-sort="ascending"' to the column header and provide functionality to change 'aria-sort' from 'ascending' to 'descending' when user click on the column header.

MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713803
